Makefile

```diff
@@ -94,7 +94,12 @@ install:
 tar: .podman-from-container
 	rm -f $(ASSET_DIR).tar.gz
 	mkdir -p $(ASSET_DIR)/usr/lib/systemd/system
+	mkdir -p $(ASSET_DIR)/usr/lib/systemd/user
 	cp -f conf/systemd/podman-restart.service $(ASSET_DIR)/usr/lib/systemd/system/podman-restart.service
+	cp -f conf/systemd/podman.service $(ASSET_DIR)/usr/lib/systemd/system/podman.service
+	cp -f conf/systemd/podman.socket $(ASSET_DIR)/usr/lib/systemd/system/podman.socket
+	cp -f conf/systemd/podman.service $(ASSET_DIR)/usr/lib/systemd/user/podman.service
+	cp -f conf/systemd/podman.socket $(ASSET_DIR)/usr/lib/systemd/user/podman.socket
```

Comment:
Why are these files copied into both the `system` and `user` directories?
I see that this is also the case within the official, fedora-based podman image but I am not sure I fully understand why. Is it so that users can modify one while still keeping the original?

Reply:
I have also just used the Debian packages as an example. I assume it is related to start podman in rootless mode.